contracts/BurnExecution.sol

```diff
@@ -95,51 +93,61 @@ contract BurnExecution is FraudProofHelpers {
         return (stateRoot, actualTxRoot, !isTxValid);
     }
 
+    function ApplyBurnExecutionTx(
+        Types.AccountMerkleProof memory _fromAccountProof,
+        Types.BurnExecution memory _tx
+    ) public view returns (bytes memory updatedAccount, bytes32 newRoot) {
+        Types.UserAccount memory account = _fromAccountProof.accountIP.account;
+
+        account.balance -= account.burn;
+        account.burn = 0;
```

Comment:
@barryWhiteHat do we want to reset burn to 0?
It's a "monthly burn" and the spec does not reset to 0. 
If we don't want to reset to zero, what do we do if we don't have enough balance the next month?

Reply:
So we don't reset to 0. if a user wants to do this they can use authorize burn with amount 0.

If they don't have enough balance the coordinator just does not include them in the batch for that month.

Reply:
Sounds good. Let me remove the reset and add a balance check.